LICENSE.md

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
We already have a LICENSE.txt in the vizro-core package. Shall we remove this one for now until we have several packages here?

Reply:
I added this so we could have a license on the top right to the repository. We also still need a license in the package, so I think it must be duplicated.

Reply:
Alright